### DISCUSSION TITLE
can i trust this on company laptop ?

### DISCUSSION THREAD

**Discussion by simonevetere** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#900] [Q&A]
can i trust this on company laptop ?

**Comment by sumeshir26** [comment]
@simonevetere Yes, of course. This app is 100% safe

**Comment by Nivet2006** [comment]
@simonevetere @sumeshir26 Just a friendly suggestion, I would recommend refraining from using any third-party software on your company laptop. You see, the laptop provided by the company is specifically meant for work purposes, and eventually, it needs to be returned. But don't worry, downloading such software won't really do any harm. So, it's best to stick to the company's guidelines and focus on the tasks at hand. Thanks for understanding!

**Comment by jaycardano** [comment]
@simonevetere  You should also check your company's policy regarding third party apps on a company computer because that could get your written up and or fired. IF you don;t if you can use. Or just ask boss he would much have you ask him than do it without and hey if your good worker he may let you do it anyway. Thanks for understanding!
